### PR TITLE
Business reviews

### PR DESCRIPTION
**Pull request by jredl-va**
- Adding basic guide for retrieving reviews for a business location. 
- Add model and api path for the api spec

Remaining: 

- [x] - Add example response to guide 

@richard-rance @Short-Hop @ndougherty-va @wfehr-va @DewaldoDev

### CODE REVIEW

File: openapi/platform/platform.yaml [hunk 1]
```diff
@@ -403,6 +403,122 @@ paths:
               properties:
                 data:
                   $ref: '#/components/schemas/businessLocations'
+  /businessLocationReviews:
```
Comment (Short-Hop) [inline openapi/platform/platform.yaml:406]: If this is going to be a Reputation API endpoint, I think it should be written within its own yaml file, placed under its own folder like `openapi > reputation > reputation.yaml`.  That is the pattern I used for Ad Intel at least.

File: docs/Guides/Reviews.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+tags: [reviews]
+---
+
+# Reviews
+
+A review of an item - for example, of a restaurant, movie, or store.
+
+## Common Actions
+
+### View reviews for a business location
```
Comment (ndougherty-va) [inline docs/Guides/Reviews.md:11]: Do we want to consider using entity so we are able to expand these APIs for service area businesses and professionals?

File: docs/Guides/Reviews.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+To view the reviews for a business location make the following request filling in the business location ID and and an [access token](../Authorization/2-legged-oauth/UsingAServiceAccount.md). This example filters the list to the most recent 2 reviews:
```
Comment (ndougherty-va) [inline docs/Guides/Reviews.md:13]: `location ID and and an`
typo `and and`

File: openapi/reputation/reputation.yaml [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+info:
+  title: Reputation REST APIs
+  version: '1.0'
+  description: ''
+servers:
+  - url: 'https://prod.apigateway.co/products/reputation'
+    description: Production
+  - url: 'https://demo.apigateway.co/products/reputation'
+    description: Demo
+  - url: '{local}/products/adintel'
```
Comment (ndougherty-va) [inline openapi/reputation/reputation.yaml:11]: ad intel

File: openapi/reputation/reputation.yaml [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+                      self:
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        description: Provides a link back to the first page of results
+                        format: uri
+                      next:
+                        type: string
+                        description: The URI at which the next batch of purchases can be gotten from
```
Comment (ndougherty-va) [inline openapi/reputation/reputation.yaml:119]: purchases -> reviews

I want to take a moment to say that I definitely am not (am) copy and pasting as well....maybe I should c+p the purchases endpoint

File: openapi/reputation/reputation.yaml [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+        publishedAt:
+          description: Date of first broadcast/publication
+          format: date-time
+          type: string
+        url:
+          description: The URL to the review
+          type: string
+        comments:
+          type: array
+          description: 'Comments, typically from users.'
```
Comment (ndougherty-va) [inline openapi/reputation/reputation.yaml:170]: quotes?

**Comment by wfehr-va** [comment]
You'll want to be merging this into the `products` branch, not `master`.

**Comment by wfehr-va** [comment]
In general, we want to match JSONAPI specs by doing models in the form of
```
{
    type:
    id:
    attributes: {}
    relationships: {}
}
```

image

image

instead of this:

image

File: openapi/reputation/reputation.yaml [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        A review of an item - for example, of a restaurant, movie, or store.
+      x-tags:
+        - Business Location Reviews
+      properties:
+        id:
+          type: string
+          example: RVW-4EF5BECE03D24D85AB4045097430A9C0
```
Comment (wfehr-va) [inline openapi/reputation/reputation.yaml:148]: Do we have any required fields in this model?

**Pull request merged**